### PR TITLE
RAML and JSON Schema multiple interface

### PR DESCRIPTION
**Pull request by ghost**
RAML files for generating RAML and JSON Schema interfaces.

https://issues.folio.org/browse/RMB-253

### CODE REVIEW

File: ramls/jsonSchemas.raml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+documentation:
+  - title: JSON Schema API
+    content: This documents the API calls that can be made to get JSON Schemas of a module defined by X-Okapi-Module-Id header
+
+types:
+  errors: !include ../schemas/errors.schema
+
+/_/jsonSchemas:
+  get:
+    description: Get JSON Schema names for the given module defined by header X-Okapi-Module-Id
```
Comment (julianladisch) [inline ramls/jsonSchemas.raml]: Explain the two cases in the description: List all schema names, get a schema for a specific path.
Can you add an example for the list of schema names including a schema in a subdirectory?

File: ramls/ramls.raml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+documentation:
+  - title: RAML API
+    content: This documents the API calls that can be made to get RAML of a module defined by X-Okapi-Module-Id header
+
+types:
+  errors: !include ../schemas/errors.schema
+
+/_/ramls:
+  get:
+    description: Get RAMLs for the given module defined by header X-Okapi-Module-Id
```
Comment (julianladisch) [inline ramls/ramls.raml]: Explain the two cases in the description: List all raml paths, return a raml file at a specific path.

File: ramls/ramls.raml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      path:
+        displayName: RAML Path
+        type: string
+        description: Path to specific RAML
+        example: tenant.raml
+        required: false
+    responses:
+      200:
+        description: "RAML"
+        body:
```
Comment (julianladisch) [inline ramls/ramls.raml:30]: Add an example for a list of raml paths including a raml in a subdirectory.

**Review by adamdickmeiss — APPROVED** [review]
Perhaps I would not have chosen the slash underscore slash prefix for this, because it's not related to Okapi at all. The tenant interface is.. But I understand that from an API point-of-view it's sort of the same.

**Comment by ghost** [comment]
@adamdickmeiss would you like I remove the underscore prefix?

**Pull request merged**